_drafts/monitoring-battery-life.md

```diff
@@ -0,0 +1,500 @@
+
+One of the most common and simplest ways to measure the current capacity within a battery, or the SoC of a device, is by measuring the voltage of the battery during a known and consistent power draw, but it's not the only way.
+
+#### Coulomb Counting with Fuel Gauges
+
+At Pebble, we had a fuel gauge on one of the watches. A fuel gauge is a nifty hardware component that can indicate the battery’s SoC and health. It can understand the battery’s current capacity in two ways: measuring the voltage from the battery and learning the patterns over time, and also Coulomb counting, which measures how much current passes in and out of the battery over time.
+
+For devices with large batteries, such as phones, e-mobility, cars, etc., fuel gauges are probably the way to go. They are reliable, and you can hand off the difficult task of measuring the current battery’s capacity to a device that was built to measure it.
+
+So with all the praise of fuel gauges, why spend so much time talking about using voltage to measure a battery’s capacity? Because at Pebble, we were unable to use the fuel gauge, as it consumes more power than we would have liked. There are thousands of products out there that might run into the same issues, especially as people are wanting to build sensors and IoT devices that last months and years on small batteries.
+
+If you happen to be lucky enough to be able to use a fuel gauge and happen to be using Zephyr RTOS, definitely check out their new Fuel Gauge API[^zephyr_fuel], announced recently in Zephyr v3.3.
+
+#### State of charge with voltage
+
+This is most commonly what companies have, as it’s very simple to get from the battery system and doesn’t consume any extra power. The problem with only tracking the voltage is that it’s not easily human-understandable, does not increase or drop linearly, and will change under operating conditions, as mentioned above.
+
+<p align="center">
+  <img width="400" src="{% img_url monitoring-battery-life/battery-curve.png %}" alt="battery-curve" />
+</p>
+
+We need something better if possible.
+
+#### State of charge with voltage and battery curve
+
+One thing that can be done to help convert voltage to a percentage is to come up with a battery curve. A battery curve is simple: it's a map between a battery's voltage and the relative percentage that likely pertains to that voltage. Products also usually have both a charge and a discharge curve.
+
+A battery curve is what companies will have after they have some form of understanding of their battery properties and have spent some time. It is more easily understood by customer support teams and engineers that aren’t directly involved with the battery subsystem. That isn’t to say that it
```

Comment:
`have spent some time`, not sure what should follow this. `have spent some time using the battery`?

Reply:
Oof, fixed.